### PR TITLE
Email updates

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
- No more purple backgrounds throughout. Restyles emails so that they're more likely to remain legible in the face of email client quirks.
- A more better button.
- Adds Fira Sans for those clients that observe these things.
- Prevents main content area from spanning the entire width of the email in Outlook. 
- Adds copy to verification email (still waiting on What To Do blurbs for breach reports).

### CODE REVIEW

File: views/email/layouts/default_email.hbs [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
     <title>Firefox Monitor</title>
     <style>
+
        body {
         text-align: center;
-        font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+        font-family: 'Fira Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
         font-weight: 400;
+        color: #333333;
+        background: #ededf0;
```
Comment (pdehaan) [inline views/email/layouts/default_email.hbs]: Will this fix #488?

File: views/email/layouts/default_email.hbs [hunk 2]
```diff
@@ -38,51 +43,39 @@
...
+      td {
+        border-collapse: collapse;
+        border-width: 0px;
+      }
+
     </style>
   </head>
-  <body style="padding: 0px; margin: 0px; width: 100%; height: 100%; line-height: 22px; font-famly: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif; color: #ffffff;" >
-    <table width="100%" height="100%" border="0" cellspacing="0" cellpadding="0" style="padding-bottom: 30px;">
+  <body bgcolor="#ededf0" style="background-color: #ededf0; padding-left: 0px; padding-right: 0px; padding-top:30px; padding-bottom: 50px; margin: 0px; width: 100%; height: 100%; line-height: 22px; font-famly: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif; color: #333333;" >
```
Comment (pdehaan) [inline views/email/layouts/default_email.hbs]: Does the `font-family` here need "Fira Sans" as well (see L12 above)?

```css
      body {
        text-align: center;
        font-family: 'Fira Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
        font-weight: 400;
        color: #333333;
        background: #ededf0;
      }
```

File: views/email/report.hbs [hunk 3]
```diff
@@ -20,43 +20,46 @@
...
-          {{> report_password_tips SERVER_URL=../SERVER_URL}}
-        {{/eachFromTo}}
+    {{else}}
+      {{#eachFromTo TIPS 0 3 }}
+        {{> report_password_tips SERVER_URL=../SERVER_URL}}
+      {{/eachFromTo}}
     {{/if}}<!--unsafeBreachesForEmail-->
   </table>
-  <table border="0" cellpadding="0" cellspacing="0" width="100%" style="padding-top: 20px; padding-bottom: 20px; margin: 0px;">
+  <table border="0" cellpadding="0" cellspacing="0" width="100%" style="padding-top: 20px; padding-bottom: 20px; margin: 0px; center;">
```
Comment (pdehaan) [inline views/email/report.hbs]: This feels "off". Not sure what should be "center"ed and I believe we're missing a rule name.

File: views/email/report.hbs [hunk 4]
```diff
@@ -20,43 +20,46 @@
...
+            <p class="body-copy" style="color: #333333; font-size: 15px;">
+              Your email appears in our database of compromised accounts. Read about each one and learn what you can do to protect your passwords. Don't forget to sign up for Firefox Monitor with all the email addresses you use for online accounts.
+            </p>
+          {{/if}}
+        </td>
+      </tr>
+      {{#each unsafeBreachesForEmail.websiteBreaches}}
+        {{> email_single_breach SERVER_URL=../SERVER_URL}}
+      {{/each}}
+      {{#ifCompare unsafeBreachesForEmail.spamLists.length ">" 0}}
```
Comment (pdehaan) [inline views/email/report.hbs]: Do we have (or need) some copy explaining what a "spam list" is?

File: views/email/layouts/default_email.hbs (hunk 1 above)
Comment (lesleyjanenorton) [reply views/email/layouts/default_email.hbs]: Uncertain but hopeful.

File: views/email/report.hbs (hunk 4 above)
Comment (lesleyjanenorton) [reply views/email/report.hbs]: Next PR removes the spam lists otherwise we would be needing it.

File: hibp.js [hunk 5]
```diff
@@ -86,8 +86,8 @@ const HIBP = {
   },
 
   async getUnsafeBreachesForEmail(sha1, allBreaches) {
-    const allFoundBreaches = await this.getBreachesForEmail(sha1, allBreaches, true);
-
+    // const allFoundBreaches = await this.getBreachesForEmail(sha1, allBreaches, true);
```
Comment (groovecoder) [inline hibp.js]: ?! This change will make `getUnsafeBreachesForEmail` return `allBreaches` for any `sha1` passed in?!

**Comment by groovecoder** [comment]
Looks like some `allFoundBreaches` debug logic was left in for testing. Otherwise this looks good.

**Comment by groovecoder** [comment]
Closing in favor of #508

**Pull request closed without merging**